avocado/plugins/run.py

```diff
@@ -85,6 +85,10 @@ def configure(self, parser):
                             default='off', help='Keep job temporary files '
                             '(useful for avocado debugging). Defaults to off.')
 
+        parser.add_argument('--force-execution', choices=('on', 'off'),
```

Comment:
From my side it's `skip-missing` or `ignore-missing` or even more verbose `ignore-missing-references` or `ignore-missing-tests`.